app/views/cookies/_banner.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-cookie-banner js-cookie-banner" role="region" aria-label="Cookies on CORE">
+  <div class="govuk-cookie-banner__message govuk-width-container js-cookie-banner__form">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-cookie-banner__heading govuk-heading-m">Cookies on CORE</h2>
+
+        <div class="govuk-cookie-banner__content">
+          <p class="govuk-body">We use some essential cookies to make this service work.</p>
+          <p class="govuk-body">We’d like to use analytics cookies so we can understand how you use the service and make improvements.</p>
+        </div>
+      </div>
+    </div>
+
+    <%= form_with url: cookies_path, class: "js-cookie-form", method: :put do |f| %>
```

Comment:
Ideally this shouldn't take you off the current page when you submit. Do you know why it seems to be doing that?

Reply:
The entire cookie banner script appears to have gone missing in a rebase 😱 

Have it locally, will add back in.